src/pages/ConfirmPage.js

```diff
@@ -1,18 +1,76 @@
 import React, { Component } from 'react';
-import { Grid, Row, Col, Alert } from 'react-bootstrap';
-import { Navbar } from '../components'
-
-const ConfirmPage = () => {
-  return (
-    <Grid fluid>
-      <Navbar />
-      <Row>
-        <Col md={8} mdOffset={2}>
-          <Alert bsStyle="success">Votre commande est validée !</Alert>
-        </Col>
-      </Row>
-    </Grid>
-  )
+import { Grid, Row, Col, Alert, Button} from 'react-bootstrap';
+import { Navbar, Cart } from '../components';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { withRouter, Redirect } from 'react-router-dom';
+import moment from 'moment';
+import { resetCheckout } from "../actions/index";
+
+
+moment.locale('fr');
+
+
+class ConfirmPage extends Component {
+
+  onClick() {
+    this.props.actions.resetCheckout();
```

Comment:
I would say that `this.props.actions.resetCheckout` can be simpler into `this.props.resetCheckout`
(given also the more sugar syntax used in my comment below)

Reply:
Does not seem to work (throw `undefined`)